views/images.html

```diff
@@ -37,13 +37,19 @@
 			{{ startImage.caption }}
 		</span>
 		<span class="q-imageslider-description__author {% if isWide %}q-imageslider-description__author--wide{% endif %} s-font-note-s s-color-gray-7">
-			{%- if startImage.credit.text %}
-				{%- if startImage.credit.link.url and startImage.credit.link.isValid %}
-					<a class="q-imageslider-description__author__url" href="{{ startImage.credit.link.url }}" target="blank" rel="noopener noreferrer">{{ startImage.credit.text }}</a>
-				{%- else %}
-					{{ startImage.credit.text }}
-				{%- endif %}
-			{%- endif %}
+			{% if startImage.credit %}
```

Comment:
Here, too, I would simplify the complexity of the if condition and then work with new lines for a better code reading
```
			{% if startImage.credit and startImage.credit.text %}
				{% if startImage.credit.link
					and startImage.credit.link.url
					and startImage.credit.link.isValid
						%}
```

Reply:
My reason for those unsightly if-statements was that I'm not quite sure, if nunjucks does short-circuiting.